tests/DocumentationExamplesTest.php

```diff
@@ -925,6 +925,41 @@ public function testExample_55_58()
         $this->assertInventoryCount(0);
     }
 
+    public function testExample_59_61()
+    {
+        $db = new Database($this->manager, $this->getDatabaseName());
+
+        // Start Changestream Example 1
+        $cursor = $db->inventory->watch();
+        $cursor->next();
+        $current = $cursor->current();
+        // End Changestream Example 1
+
+        $this->assertNull($current);
+
+        // Start Changestream Example 2
+        $cursor = $db->inventory->watch([], ['fullDocument' => \MongoDB\Operation\ChangeStreamCommand::FULL_DOCUMENT_UPDATE_LOOKUP]);
+        $cursor->next();
+        $current = $cursor->current();
+        // End Changestream Example 2
+
+        $this->assertNull($current);
+        $db->inventory->insertOne(['x' => '1']);
```

Comment:
While it makes no functional difference, `'1'` can probably be an integer literal. If you do want to use a string, something like `'foo'` would be more consistent.